lib/tasks/access_tokens.rake

```diff
@@ -0,0 +1,14 @@
+namespace :access_tokens do
+  desc "Add a pre-generated token, using the SHA256 hex digest"
+  task :insert, %i[token_digest] => :environment do |_, args|
+    usage_message = "usage: rails access_tokens:insert[<token_digest>]".freeze
+    abort usage_message unless args[:token_digest]&.match?(/^[a-f0-9]+$/)
+
+    access_token = AccessToken.create!(
+      description: "inserted with `rails access_tokens:insert`",
+      owner: ENV["USER"],
```

Comment:
Just looking dev where I presume you've run this and the user is listed as `ruby`. The user is intended to be the name of the developer (or app such as forms-runner) who owns the key and I think that's important to maintain.
```
bash-3.2$ forms data_api -d forms-api -s "select DISTINCT owner from access_tokens where description LIKE 'inserted with%';" | jq '.records[]' -r
{
  "owner": "ruby"
}
```

Reply:
That's a fair point. I wasn't sure how to get the user information in, I thought that maybe it didn't matter so much, but if you think it does I'll have another think about it!

Reply:
yeah the reasons it's important are that we use the owner in our logs to log who made the API call (and it'll be important for general access auditing): https://github.com/alphagov/forms-api/blob/368abfdbf6b6cc23a13efff51b8a078118491fb4/app/controllers/application_controller.rb#L27

And we'll need to use it for management of those keys (for example when someone leaves) and to notify people for key rotation etc

Reply:
I've changed the task slightly to add the owner as a parameter as well as the token digest. 

I did spend a while looking into whether it was possible to get the AWS user who started the EC2 task out of the [ECS metadata](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-metadata-endpoint.html) somehow, but I couldn't find anything straightforward.